src/player/health.rs

```diff
@@ -1,5 +1,12 @@
-use crate::game_over::*;
-use crate::player::*;
```

Comment:
The rusty way of importing things is to use fully qualified paths, the `prelude`s some crates use is the exception. Wildcard imports are also used in `test` modules.